tests/acceptance/features/bootstrap/TestingAppContext.php

```diff
@@ -951,49 +951,6 @@ public function theAdministratorHasCreatedFollowingLocks(TableNode $table) {
 		}
 	}
 
-	/**
-	 * Expires last created share using the testing API
-	 *
-	 * @return void
-	 */
-	public function expireLastCreatedUserShare() {
-		$adminUser = $this->featureContext->getAdminUsername();
-		$share_id = $this->featureContext->getLastShareId();
-		$response = OcsApiHelper::sendRequest(
-			$this->featureContext->getBaseUrl(),
-			$adminUser,
-			$this->featureContext->getAdminPassword(),
-			'POST',
-			$this->getBaseUrl("/expire-share/{$share_id}"),
-			[],
-			$this->featureContext->getOcsApiVersion()
-		);
-		$this->featureContext->setResponse($response);
-	}
-
-	/**
-	 * @Given the administrator has expired the last created share using the testing API
-	 *
-	 * @return void
-	 */
-	public function theAdministratorHasExpiredTheLastCreatedShare() {
-		$this->expireLastCreatedUserShare();
-		Assert::assertSame(
-			200,
-			$this->featureContext->getResponse()->getStatusCode(),
-			"Request to expire last share failed."
-		);
-	}
-
-	/**
-	 * @When the administrator expires the last created share using the testing API
```

Comment:
looks like this step was not duplicate. According to the drone run https://drone.owncloud.com/owncloud/testing/1471/14/9
```

--- FeatureContext has missing steps. Define them with these snippets:
--
874 | &nbsp;
875 | /**
876 | * @When the administrator expires the last created share using the testing API
877 | */
878 | public function theAdministratorExpiresTheLastCreatedShareUsingTheTestingApi()
879 | {
880 | throw new PendingException();
881 | }

```

Reply:
core PR https://github.com/owncloud/core/pull/38805 has now been merged. I will restart CI.